docs/building-modules/invariants.md

```diff
@@ -1,5 +1,86 @@
 # Invariants
 
-## What is an invariant
+## Pre-requisite Reading
 
-## Invariant Registry
\ No newline at end of file
+- [Keepers](./keeper.md)
+
+## Synopsis
+
+An invariant is a property of the application that should always be true. An `Invariant` is a function that checks for a particular invariant. These functions are useful to detect bugs early on and act upon them to limit the potential consequences (e.g. by halting the chain). They are also useful in the development process of the application to detect bugs via simulations. 
+
+- [Implementing `Invariant`s](#implementing-invariants)
```

Comment:
why the link to something directly beneath?

Reply:
It'll be turned in side menu on the website, for navigation within the page. 

We have table of content for the doc on each doc